pkg/network/master/master.go

```diff
@@ -146,6 +146,27 @@ func Start(networkConfig osconfigapi.MasterNetworkConfig, networkClient networkc
 		return err
 	}
 
+	var nodes []kapi.Node
+	var subnets []networkapi.HostSubnet
+	if nodeList, err := master.kClient.Core().Nodes().List(metav1.ListOptions{}); err == nil {
+		nodes = nodeList.Items
+	}
+	if subnetList, err := master.networkClient.Network().HostSubnets().List(metav1.ListOptions{}); err == nil {
+		subnets = subnetList.Items
+	}
+
+	for _, subnet := range subnets {
+		for i, node := range nodes {
```

Comment:
Ignore the subnet if it has  "pod.network.openshift.io/assign-subnet" annotation

Reply:
I know we talked about this but in working on this BZ I found that a subnet does not maintain that annotation. In pkg/network/master/subnets.go if the watch sees that annotation it deletes the user created subnet and the system allocates one for the same host